packages/contrast-colors/index.js

```diff
@@ -36,6 +36,20 @@ function cArray(c) {
   return new Array(L, U, V);
 }
 
+function removeDuplicates(originalArray, prop) {
```

Comment:
So this is a little contrived, but I think I see what you're doing here (checking if the prop is duplicated on a child array). It's still `O(N)`, so it's fine :)